cedar-policy-validator/src/type_error.rs

```diff
@@ -262,6 +259,7 @@ pub enum TypeErrorKind {
     /// The typechecker could not conclude that an access to an optional
     /// attribute was safe.
     #[error("unable to guarantee safety of access to optional attribute {}", .0.attribute_access)]
+    #[diagnostic(help("try testing for the attribute with `e has a && ..`"))]
```

Comment:
we could perhaps make this suggestion better by using the real attribute name instead of `a` (or even the real LHS instead of `e`)

Reply:
The attribute names are already available on the `AttributeAccess` struct, so I can do something with those.